go.mod

```diff
@@ -260,3 +259,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/pulumi/pulumi-kubernetes/provider/v4 => github.com/pulumi/pulumi-kubernetes/provider/v4 v4.0.0-20240930210339-a5873e0cdf83
```

Comment:
Will remove from this PR once https://github.com/pulumi/pulumi-kubernetes/pull/3237 lands.